Client/Frontend/Home/RemoteTabsPanel.swift

```diff
@@ -82,28 +84,46 @@ class RemoteTabsPanel: UITableViewController, HomePanel {
             return
         }
 
-        // Otherwise, fetch the tabs cloud
-        self.profile.getClientsAndTabs().uponQueue(dispatch_get_main_queue()) { result in
-            self.refreshControl?.endRefreshing()
-
+        self.profile.getCachedClientsAndTabs().uponQueue(dispatch_get_main_queue()) { result in
```

Comment:
@rnewman I've updated the logic to grab the cached tabs, then try to perform a sync if needed, and if the sync fails, the stale tabs will be shown instead of the error state. How does this look to you?